### ISSUE TITLE
"Edit Toolbar" does not stay open

### ISSUE THREAD

**Issue by dreadbalaur** [nomacs/nomacs#1053]
**Describe the bug**
The Edit Toolbar is hidden by default and I keep having to toggle it on whenever I launch nomacs. There is no clear option in the setting to keep it always visible. 

**To Reproduce**
1. Launch nomacs
2. Right-click and turn on the Edit Toolbar.
3. Close nomacs.
4. Launch nomacs again and the Edit Toolbar is hidden again. 

**Expected behavior**
Have the Edit Toolbar visible by default. This was the case for older nomacs builds. 

**Desktop (please complete the following information):**
 - OS: Windows 10 22H2
 - Version: 3.17.2297 x64

**Additional context**
If this is an expected behavior of the new build then you can move this to "Feature request".

**Comment by leejuyuu** [comment]
Hello @dreadbalaur , thanks for reporting. Could you provide the version of the older build you mentioned?

**Comment by dreadbalaur** [comment]
Sorry, did not get notified on the reply. 

The older build was downloaded from the website in 2020 before I became aware of the github page - so potentially 3.15?

**Comment by leejuyuu** [comment]
Do you still have the older version? If you do, please check the version in `? > About nomacs`. This will greatly help us to track down whether this is a regression. Thanks!

**Comment by dreadbalaur** [comment]
Update: it was 3.16.1709.

However, after rolling it back, the issue persisted. But I eventually solved my issue after fiddling around.

Editor -> AppSettings -> hideallpanels = false. This should technically be the same behavior as "Hide All Panels" in the Panels tab, but my first attempt with toggling it through the tab reveals other panels as well (overview, histogram, and metadata panels). Once I manually changed the value in the editor and re-updated the program to 3.17.2297, the behavior is fixed: edit toolbar remains open when I restart the program, and toggling "Hide All Panels" does not reveal extra panels.

I can't test for other scenarios to be more certain, but for anyone else having this issue: toggle "Hide All Panels" in the Panels tab (or press F) after you unhide the Edit Toolbar. If it still does not behave as expected, adjust the value directly in the Editor.
